### ISSUE TITLE
Remove "Loading..." text

### ISSUE THREAD

**Issue by djalmajr** [xcarpentier/rn-pdf-reader-js#77]
How to remove the "Loading..." page  from PDF viewer?

<img src="https://user-images.githubusercontent.com/1858147/67150990-6892a280-f295-11e9-9d5e-e2147f89a271.png" width="360">

**Comment by Kamuza** [comment]
same here..

**Comment by iWexter** [comment]
same...

**Comment by djalmajr** [comment]
As a workaround I did the following in `index.js`

```javascript
function viewerHtml(base64: string): string {
  ...
    <script type="text/javascript" src="bundle.js"></script>
    <script>
      // Remove "Loading..." text
      var p = document.querySelector("p");
      p.parentNode.parentNode.removeChild(p.parentNode);
    </script>
  ...
}
```

**Comment by iWexter** [comment]
I did the pending pull request on all my project

**Comment by xcarpentier** [comment]
Should be fixed at `v0.5.0`.
